app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.kt

```diff
@@ -45,6 +45,10 @@ class LoginFragment private constructor() : WolmoFragment<FragmentLoginBinding,
         binding.email.error = getString(R.string.login_alert_bad_email)
     }
 
+    override fun showLoginError() {
+        println("Log In Error - Invalid Credentials")
```

Comment:
I don't think it's a good idea to leave a `println()`. If you want to print an error you should use Log.e for errors, or any other kind of log you want. Any ways, instead of logging the error (lets think we don't need that in this case), you should tell the user that an error happened.